lib/src/values/logic_value.dart

```diff
@@ -761,6 +767,20 @@ abstract class LogicValue {
       // ignore: avoid_dynamic_calls
       _doCompare(other, (a, b) => (a >= b) as bool);
 
+  /// Power operation
+  ///
+  /// Both inputs [a] and [b] are either of type [int] or [BigInt].
+  /// Returns [a] raise to the power [b] of same input type.
+  dynamic _powerOperation(dynamic a, dynamic b) {
+    if (a is BigInt) {
+      // ignore: avoid_dynamic_calls
```

Comment:
We can make some casts on `a` and `b` to highlight the assumption that either both are `BigInt` or both are `int`.  Then we can avoid these lint ignores.